news/2020-11-19-better-protecting-players-from-abuse.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Better protecting players from abuse"
+date: 2020-11-19 09:00:00 +0000
+---
+
```

Comment:
header image missing, probably need to bother flyte for one or use a generic from /wiki/shared/news/banners

Reply:
Pretty sure it'll just fallback to a generic one (see https://osu.ppy.sh/home/news/2016-10-15-gmt-applications-now-open as an example), unless you have a more appropriate suggestion.

Reply:
i suppose that one will do